extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/NettyProcessor.java

```diff
@@ -171,10 +171,16 @@ NativeImageConfigBuildItem build(
         if (QuarkusClassLoader.isClassPresentAtRuntime("io.netty.buffer.UnpooledByteBufAllocator")) {
             builder.addRuntimeReinitializedClass("io.netty.buffer.UnpooledByteBufAllocator")
                     .addRuntimeReinitializedClass("io.netty.buffer.Unpooled")
-                    .addRuntimeReinitializedClass("io.vertx.core.http.impl.Http1xServerResponse")
                     .addRuntimeReinitializedClass("io.netty.handler.codec.http.HttpObjectAggregator")
```

Comment:
I think they were added there by mistake and should be moved to the VertxCoreProcessor.